superdesk/editor_utils.py

```diff
@@ -349,8 +349,16 @@ def render(self):
                 block.setdefault("entityRanges", [])
                 block.setdefault("inlineStyleRanges", [])
                 if block.get("text"):
-                    block["text"] = "".join(ch for ch in block["text"] if unicodedata.category(ch) != "Cc")
+                    normalized_text = []
+                    for ch in block["text"]:
+                        if ch == "\n":
+                            normalized_text.append("\u2028")  # Use Unicode LINE SEPARATOR
+                        elif unicodedata.category(ch) != "Cc":
```

Comment:
what are you trying to filter out here? it might also remove tabs if I got it right

Reply:
The filtering is intended to remove control characters (Unicode category C) that can break XML rendering in lxml. Without it, certain characters in the content (even invisible ones) were causing errors like ValueError: All strings must be XML compatible. We do allow valid formatting characters like \n (handled via LINE SEPARATOR) and can preserve tabs if needed. This ensures that normal text and Unicode (like qualité) render correctly without breaking the HTML export.

Reply:
I see